docs/lazydocs.generation.md

```diff
@@ -1,14 +1,14 @@
 <!-- markdownlint-disable -->
 
-<a href="https://github.com/ml-tooling/lazydocs/blob/main/src/lazydocs/generation.py#L0"><img align="right" style="float:right;" src="https://img.shields.io/badge/-source-cccccc?style=flat-square"></a>
+<a href="https://github.com/ml-tooling/lazydocs/blob/main/src/lazydocs/generation.py#L0"><img align="right" style="float:right;" src="https://img.shields.io/badge/-source-cccccc?style=flat-square" ></a>
```

Comment:
for some reasons, it seems that it has not worked here 😅 does it have to run again maybe?

Reply:
Should be fine now.